src/flamegraph/mod.rs

```diff
@@ -277,60 +277,6 @@ impl Default for Direction {
     }
 }
 
-struct FrameAttributes<'a> {
-    title: &'a str,
-    id: Option<&'a str>,
-    class: Option<&'a str>,
-    g_extra: Option<&'a Vec<(String, String)>>,
-    href: Option<&'a str>,
-    target: &'a str,
-    a_extra: Option<&'a Vec<(String, String)>>,
-}
-
-fn override_or_add_attributes<'a>(
-    title: &'a str,
-    attributes: Option<&'a attrs::FrameAttrs>,
-) -> FrameAttributes<'a> {
-    let mut title = title;
-    let mut id = None;
-    let mut class = None;
-    let mut g_extra = None;
-    let mut href = None;
-    let mut target = "_top";
```

Comment:
I think I missed it -- where is this default now kept?

Reply:
Yeah, good catch. I noticed this right after I submitted the PR. I'll fix it.